### PR TITLE
Da/subs

### PR DESCRIPTION
**Pull request by tadad**

### CODE REVIEW

File: sur/subscriber.hoon [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/+  *mip
+|%
++$  subscriber-action
+  $%  [%sub =ship agent=@tas =path]
+      [%unsub =ship agent=@tas =path]
+      [%clear =ship agent=@tas =path]
+  ==
+::  map of ship to agent name to subscription path to signs
++$  facts  (mip [ship term] wire (qeu sign:agent:gall)) :: XX path?
```
Comment (hosted-fornet) [inline sur/subscriber.hoon:9]: This is cool: didn't know about `mip`

File: app/subscriber.hoon [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  |=  =old=vase
+  ^-  (quip card _this)
+  =/  old-state  !<(state-0 old-vase)
+  `this(state old-state)
+++  on-poke
+  |=  [=mark =vase]
+  ^-  (quip card _this)
+  ?>  =(mark %subscriber-action)
+  =/  act  !<(subscriber-action vase)
+  =/  paf=path  [(scot %p ship.act) agent.act path.act]
```
Comment (hosted-fornet) [inline app/subscriber.hoon:32]: style nit: here and below can do

```
[agent path]:act
```

**Review by hosted-fornet — APPROVED** [review]
Other then some optional style nits, LGTM

File: sur/subscriber.hoon (hunk 1 above)
Comment (tadad) [reply sur/subscriber.hoon:9]: yeah it's useful - was thinking about making it a `(map ship (map term (map wire (qeu sign))))` but seemed a little overkill

**Pull request merged**